prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -51,12 +48,18 @@ def __init__(self, config: Config) -> None:
         self.password = config.redfish_password
         self.timeout = config.redfish_client_timeout
         self.max_retry = config.redfish_client_max_retry
-        self.redfish_obj: HttpClient = self._get_redfish_obj()
+        self.redfish_obj: HttpClient = None
```

Comment:
**nitpick:** 

Missing typing

```
self.redfish_obj: Optional[HttpClient] = None
```

Reply:
Good catch, I've fixed it :)

Reply:
After thinking about it a bit, I've modified the code like so:
```python
# redfish.py

    def __init__(self, config: Config) -> None:
        """Initialize values for class."""
        self.host = config.redfish_host
        self.username = config.redfish_username
        self.password = config.redfish_password
        self.timeout = config.redfish_client_timeout
        self.max_retry = config.redfish_client_max_retry
        self.redfish_obj: HttpClient
```

I'm not setting an initial value for `redfish_obj` now and setting the type only to `HttpClient`. Because in case the redfish_client login fails, we're anyway throwing an exception and raising that error.

What are your thoughts on this?